runtime/common/src/lib.rs

```diff
@@ -319,6 +319,7 @@ pub enum ProxyType {
 	CancelProxy,
 	Governance,
 	Auction,
+	DexLiquidity,
```

Comment:
this needs to be the last variant to avoid breaking changes on the encoding format